examples/offscreen-canvas.js

```diff
@@ -120,6 +131,8 @@ worker.addEventListener('message', (message) => {
       );
     });
     image.src = event.data.src;
```

Comment:
Please include this change, the window.event is deprecated.
```suggestion
    image.src = message.data.src;
```